package.json

```diff
@@ -33,18 +35,24 @@
         "url": "https://github.com/betagouv/api-subventions-asso/issues"
     },
     "homepage": "https://github.com/betagouv/api-subventions-asso#readme",
+    "workspaces": [
+        "packages/*"
+    ],
     "dependencies": {
         "husky": "^7.0.4",
         "ts-node": "^10.4.0",
-        "lerna": "^5.5.1"
+        "lerna": "^7.1.5"
```

Comment:
J'ai déplacé lerna dans les deps car techniquement pour la package global on en a besoin pour fonctionner ! ça regle bien le souci de lerna not found